nipype/interfaces/io.py

```diff
@@ -2808,14 +2813,31 @@ def __init__(self, infields=None, **kwargs):
 
     def _list_outputs(self):
         # Version resilience
+        import bids
+        from packaging import version
         try:
             from bids import BIDSLayout
         except ImportError:
             from bids.grabbids import BIDSLayout
+
+        pybids_ver = version.parse(bids.__version__)
         exclude = None
         if self.inputs.strict:
             exclude = ['derivatives/', 'code/', 'sourcedata/']
-        layout = BIDSLayout(self.inputs.base_dir, exclude=exclude)
+
+        if pybids_ver < version.parse('0.5'):
```

Comment:
Maybe you can include something like this in my PR? My PR is  more general update to 0.7, and includes some extra functionality / updates

Reply:
Although now that I think about it,  I wouldn't worry too much about backwards compatibility. I would make 0.7 the hard requirement. The default behavior of my PR is to look for derivatives in `/derivatives`, but optionally other paths can be added.